pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.3
```

Comment:
Please revert the lock. We could update it in another PR.

Reply:
sure :-)